tests/core/test_resolving.py

```diff
@@ -13,3 +13,22 @@
 # along with Blue Brain Nexus Forge. If not, see <https://choosealicense.com/licenses/lgpl-3.0/>.
 
 # Placeholder for the test suite for selection of target or resolver from input and configuration.
+
+from kgforge.core import KnowledgeGraphForge
```

Comment:
Should'nt this go to tests/core/test_forge.py where forge init functions are done. test_resolving is more for the actual resolving business.

Reply:
I moved the unit test to test_forge.py and pushed the changes